python/cudf/cudf/tests/series/test_datetimelike.py

```diff
@@ -1 +1,63 @@
 # Copyright (c) 2023, NVIDIA CORPORATION.
+
+import pandas as pd
+import pytest
+
+import cudf
+from cudf import date_range
+from cudf.testing._utils import assert_eq
+
+
+@pytest.fixture(params=["ns", "us", "ms", "s"])
+def unit(request):
+    return request.param
+
+
+@pytest.fixture(
+    params=["America/New_York", "Asia/Tokyo", "CET", "Etc/GMT+1", "UTC"]
```

Comment:
I'd like to see if we can test _every_ time zone present on the system against pandas behavior for some function(s).

Reply:
Yes, this is a good solution. I think testing all timezones for a very limited subset of functions is a helpful way to make sure we're not making bad assumptions about the timezone database structure which may change over time or by system.